WordPress/src/main/res/values/strings.xml

```diff
@@ -1042,9 +1042,15 @@
     <string name="passcode_turn_on">Turn PIN lock on</string>
     <string name="passcodelock_prompt_message">Enter your PIN</string>
     <string name="passcodelock_hint"></string>
+
+    <!--
+      Jetpack strings
+    -->
     <string name="jetpack_message">The Jetpack plugin is required for stats. Do you want to install Jetpack?</string>
     <string name="jetpack_message_not_admin">The Jetpack plugin is required for stats. Contact the site administrator.</string>
     <string name="jetpack_not_found">Jetpack plugin not found</string>
+    <string name="jetpack_not_connected">Jetpack plugin not connected</string>
+    <string name="jetpack_not_connected_message">The Jetpack plugin is installed, but not connected with WordPress.com. Do you want to connect Jetpack?</string>
```

Comment:
`but not connected with WordPress.com` should probably be `but not connected to WordPress.com`. Better preposition and to match the connect button in the admin. 
<img width="354" alt="screen shot 2016-04-19 at 3 56 45 pm" src="https://cloud.githubusercontent.com/assets/1435271/14655346/7597f478-0647-11e6-8f6f-c19c451eb3b5.png">